### DISCUSSION TITLE
五笔拼音输入法如何关闭拼音的简拼？

### DISCUSSION THREAD

**Discussion by TomatoFryEggs** [rime/home#853] [Q&A]
使用该https://github.com/rime/rime-wubi 方案。

比如在打cb的时候希望只出现 予（cb-j）等五笔相关提示，而不会出现 _成本_ 等词，只有在我打出 chengben 的时候才会出现 _成本_ 等词毕竟使用拼音是少数。

**Comment by lotem** [comment]
​刪除以下兩行代碼：

https://github.com/rime/rime-pinyin-simp/blob/b0e84cda02c613ebdedc127a26131b3800f45a8e/pinyin_simp.schema.yaml#L53

https://github.com/rime/rime-pinyin-simp/blob/b0e84cda02c613ebdedc127a26131b3800f45a8e/pinyin_simp.schema.yaml#L54

**Comment by TomatoFryEggs** [comment]
> ​刪除以下兩行代碼：
> 
> https://github.com/rime/rime-pinyin-simp/blob/b0e84cda02c613ebdedc127a26131b3800f45a8e/pinyin_simp.schema.yaml#L53
> 
> https://github.com/rime/rime-pinyin-simp/blob/b0e84cda02c613ebdedc127a26131b3800f45a8e/pinyin_simp.schema.yaml#L54

感谢！

**Comment by TomatoFryEggs** [comment]
除去了简拼带来的影响，发现还有权重会对词频有影响，试了下将权重调高貌似没生效，所在五笔拼音输入法中五笔和拼音的权重是由什么决定？我怎么做才能优先显示五笔词库里的反查？ @lotem
